boards/arm/nucleo_l412rb_p/arduino_r3_connector.dtsi

```diff
@@ -36,4 +36,4 @@
 };
 
 arduino_i2c: &i2c1 {};
-arduino_spi: &spi1 {};
+arduino_spi: &spi2 {};
```

Comment:
This change is not mentioned in the commit message

Reply:
Done